setup.py

```diff
@@ -22,6 +23,22 @@
     if version_parts[-1] == 'dirty':
         version += '.dev'
 
+emojis_json_file = 'ntfy/emojis.json'
+if exists(emojis_json_file):
+    print "Using existing", emojis_json_file
```

Comment:
This isn't py3k compatible (thus tests failing in jenkins). Add `from __future__ import print_function` at the top of the file and use `print('foobar')` instead of the statement version :smiley: